backend/src/litigation_api/ingestion/pipeline.py

```
@@ -0,0 +1,102 @@
+
+from ..models.domain import Case, DocketEntry, Document
+from .crlca_models import CRLCACase, CRLCADocket, CRLCADocument
+from .normalization import normalize_case, normalize_docket_entry, normalize_document
+from .client import CRLCAClient
+
+logger = logging.getLogger(__name__)
+
+SUPPORTED_DOCUMENT_TYPES = {"Complaint", "Opinion/Order", "Settlement"}
+
+class IngestionPipeline:
+    def __init__(self, s3_endpoint_url: str, s3_bucket_name: str, client: Optional[CRLCAClient] = None):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)